sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -401,6 +427,46 @@ object ScalaJSPluginInternal {
               "are running a JVM REPL. JavaScript things won't work.")
       )),
 
+      scalaJSNativeLibraries := {
+        collectFromClasspath(fullClasspath.value,
+            "*.js", collectJar = _.jsFiles,
+            collectFile = FileVirtualJSFile.relative)
+      },
+
+      jsDependencyManifests in jsManifestFilter := {
+        collectFromClasspath(fullClasspath.value,
+            new ExactFilter(JSDependencyManifest.ManifestFileName),
+            collectJar = _.jsDependencyManifests,
+            collectFile = { (file, _) =>
+              fromJSON[JSDependencyManifest](readJSON(IO.read(file)))
+            })
+      },
+
+      jsDependencyManifests := {
+        val filter = jsManifestFilter.value
+        val rawManifests = (jsDependencyManifests in jsManifestFilter).value
```

Comment:
I think we should leave the `jsManifestFilter` by inspiration of sbt's `excludeFilter`, `includeFilter` and `testFilter`.

Compare:
```scala
jsManifestFilter :=
  ManifestFilters.reinterpretResourceNames("foo")("bar" -> "1.2.1/bar.js")
// vs
jsDependenciesManifests ~= {
  _.map(ManifestFilter.reinterpretResourceNames("foo")("bar" -> "1.2.1/bar.js"))
}
```

I did not include the `_.andThen(...)` since one really shouldn't need to do this. If one depends on previously defined manifest filters, something is really off already.